languages/common-lisp/exercises/concept/sameness/.meta/example.lisp

```diff
@@ -0,0 +1,26 @@
+(defpackage sameness
+  (:use :cl)
+  (:export :robot))
+
+(in-package :sameness)
+
+(defun robot (room-id)
```

Comment:
This is quite a few cases! And quite a few with `equalp`. I think that's alright, but I'm also open to discussions about trimming that a bit as well.

Reply:
Yes, `equalp` is definitely a workhorse! I already dropped pathnames from consideration, I could also drop structures and hashtables (only really relevant to `equalp`) Hash tables might be good to drop since the example here is not a big one and does gloss over the issue of hash table test predicate....

So I'd be willing to get rid of both, or keep both - what do you think?

are there any other `equalp` cases you think i should remove?

Reply:
Hmm, seeing as structures and hash-tables are rather advanced and (presumably) will mention equality in their own exercises, I think we could probably leave those two out for the time being? With that being said, I probably wouldn't want to drop too much more than those two – I think all of the others are still very helpful and not too confusing.

Reply:
OK, will remove them.